requirements_docker.txt

```diff
@@ -0,0 +1,6 @@
+# include base requirements
```

Comment:
with an increasing number of requirements file, is it worth to put them inside a requirements folder?

Reply:
Possibly. Doing that at this moment would break https://github.com/uisautomation/iar-frontend/pull/40 so I'd rather have it as a separate story once that has landed.

Reply:
Captured this as #10.